onnx/defs/quantization/defs.cc

```diff
@@ -83,11 +93,16 @@ ONNX_OPERATOR_SET_SCHEMA(
             "y",
             "N-D full precision output tensor. It has same shape as input 'x'.",
             "tensor(float)")
+        .Attr(
+            "axis",
```

Comment:
Please update the documentation to include axis. How is axis incorporated in this formula "y = (x - x_zero_point) * x_scale" ?

Other quantization ops like QLinearMatmul which allow 1-D tensors for scale and zero point do not have axis attribute, I am trying to understand whether we need to update those ops too?

Reply:
Which documentation do I need to update specifically? I've added it as an attribute for these ops, and generally I've seen in the ONNX spec that attributes don't need to be explicitly explained.

I'm targeting only the QDQ Linear ops in this MR since these are the most general quantization operators.

Reply:
I am trying to understand why do we need axis at all? We do per channel quantization and data is in format NCHW so we know where channel is?

Reply:
For QuantizeLinear and DequantizeLinear ops, they currently only support per-tensor quantization. This MR is for expanding the definition to also include per-axis quantization.